Test/XLTest.cs

```diff
@@ -803,16 +811,16 @@ public static void TestMs2Ms3()
                 csm.BetaPeptide.ResolveAllAmbiguities();
             }
             // test parent scan (CID)
-            Assert.That(csm.MatchedFragmentIons.Count == 37);
+            Assert.AreEqual(csm.MatchedFragmentIons.Count, 32);
             Assert.That(csm.ScanNumber == 2);
 
             // test child scan (low-resolution CID, alpha peptide signature ion)
-            Assert.That(csm.ChildMatchedFragmentIons.First().Key == 6);
```

Comment:
@lonelu can you look at this? I don't feel good about this after the merge conflicts.

Reply:
It's almost like the alpha and beta peptides switched?

Reply:
After looking at this, the alpha and beta peptides switched BEFORE the merge, so this might actually be fine. Concerning that the new fragmentation caused changes, but I think the merge "Merge branch 'master' into memoryEfficientFragmentation 0644682" is OK.

Reply:
#1780 resolved some bugs in the XL search, I'm guessing that fixing those caused some scores to change